web-app/helpers/certificate-designs.ts

```diff
@@ -1,17 +1,16 @@
 // https://dev.to/sudo_overflow/diy-generating-dynamic-images-on-the-fly-for-email-marketing-h51
-import {
-  Canvas,
-  // registerFont,
-  loadImage,
-} from 'canvas';
+import { Canvas, registerFont, loadImage } from 'canvas';
 
 // TODO: Update this section:
-export const width = 1160; // width of the image
-export const height = 653; // height of the image
-// const fontFile = './fonts/Sign-Painter-Regular.ttf';
-const fontFamily = 'signpainter';
+export const width = 1080; // width of the image
+export const height = 1080; // height of the image
+const fontFile = './fonts/Manrope-VariableFont_wght.ttf';
+const fontFamily = 'Manrope, Sans Serif';
+const fontFile2 = './fonts/DMMono-Medium.ttf';
```

Comment:
@norrec99 Please don't append numbers to variable names. Please rename each variable to make them more obvious about what each is referring to.

Reply:
Done